### DISCUSSION TITLE
See logs in web app

### DISCUSSION THREAD

**Discussion by perroboc** [LizardByte/community#48] [Sunshine]
I've setup Sunshine as a windows service, and now I don't know where the logs are! It would be great to see them through the web app for easy troubleshooting :)

**Comment by ReenigneArcher** [comment]
I agree this would be a nice feature!

Some other ideas to build on top of yours.

- Add a way to download the logs from the UI
- Add a "copy" logs button to copy the logs to the clipboard
- Add the ability to write the terminal output to rotating log files

**Reply by ReenigneArcher** [reply]
There is a reference to a log file here: https://github.com/LizardByte/Sunshine/blob/70ae7a2fa9b33173eb949c86ec42a96e7b9e8c25/tools/sunshinesvc.cpp#L67

Were you ever able to find a log file?

**Comment by ReenigneArcher** [comment]
This is now included in Sunshine.
